crates/cheatnet/tests/cheatcodes/spoof.rs

```diff
@@ -113,22 +111,24 @@ impl TxInfo {
     fn deserialize(data: &[Felt252]) -> Self {
         let mut reader = BufferReader::new(data);
 
-        let version = reader.read_felt();
-        let account_contract_address = reader.read_felt();
-        let max_fee = reader.read_felt();
-        let signature = reader.read_vec();
-        let transaction_hash = reader.read_felt();
-        let chain_id = reader.read_felt();
-        let nonce = reader.read_felt();
-        let resource_bounds_len = reader.read_felt();
-        let resource_bounds = reader.read_vec_body(
-            3 * resource_bounds_len.to_usize().unwrap(), // ResourceBounds struct has 3 fields
-        );
-        let tip = reader.read_felt();
-        let paymaster_data = reader.read_vec();
-        let nonce_data_availability_mode = reader.read_felt();
-        let fee_data_availability_mode = reader.read_felt();
-        let account_deployment_data = reader.read_vec();
+        let version = reader.read_felt().unwrap();
```

Comment:
Shouldn't this also return the `Result`? Isn't it possible that the failure will crash the runner here?

Reply:
Why do we unwrap here? Would be nice to leave a comment before resolving

Reply:
Because it's in our tests only